config/environments/production.rb

```diff
@@ -111,5 +111,14 @@
   # config.active_record.database_resolver_context = ActiveRecord::Middleware::DatabaseSelector::Resolver::Session
 
   config.action_mailer.default_url_options = { host: 'https://phinance-io-abc-tech.herokuapp.com/' }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    :address => "smtp.gmail.com",
+    :port => 587,
+    :user_name => ENV["GMAIL_USERNAME"],
+    :password => ENV["GMAIL_PASSWORD"],
+    :authentication => "plain",
+    :enable_starttls_auto => true
+  }
```

Comment:
I think you don't have to put this config here since you already defined it on `environment.rb`